src/App.vue

```diff
@@ -1,19 +1,82 @@
 <template>
+  <LoaderSpinner v-if="isLoading" />
```

Comment:
It's better to move this logic inside the LoaderSpinner component. Plus, it's better to use 'v-show' because 'v-if' is deleting the component and renders it every time when condition changed.